src/methods.js

```diff
@@ -31,17 +31,27 @@ export default Base =>
       } = newState
 
       // Determine if there are Header Groups
-      const hasHeaderGroups = columns.some(column => column.columns)
+      let hasHeaderGroups = false
+      columns.forEach(column => {
+        if (column.columns) {
+          hasHeaderGroups = true
+        }
+      })
+
+      let columnsWithExpander = [...columns]
 
-      // Find the expander column which could be deep in tree of columns
-      const allColumns = _.iterTree(columns, 'columns')
-      const expanderColumn = _.getFirstDefined(allColumns)
+      let expanderColumn = columns.find(
+        col => col.expander || (col.columns && col.columns.some(col2 => col2.expander))
+      )
 
+      f (expanderColumn && !expanderColumn.expander) {
```

Comment:
this looks like a typo, should be `if` IMO

Reply:
@jbauerrfid thanks, not sure how I missed this. I'm away for awhile, maybe you would be willing to submit PR to fix this?